torch/nn/functional.py

```diff
@@ -38,7 +38,8 @@ def conv2d(input, weight, bias=None, stride=1, padding=0, dilation=1,
         groups: split input into groups, in_channels should be divisible by
           the number of groups
 
-    Examples:
+    Examples::
```

Comment:
Did you try compiling the docs and opening in the browser to see if this works? (I wonder if we should add a `..` before)
You can go to `docs` and run `make html` to compile the docs.